### ISSUE TITLE
[Feature request]I have an idea

### ISSUE THREAD

**Issue by lxl19920401** [mengshukeji/Luckysheet#725]
![无标题](https://user-images.githubusercontent.com/28803441/127594997-f950fe52-4a28-481e-9626-ae9435fef3e5.png)
![无标题2](https://user-images.githubusercontent.com/28803441/127595020-8acc83c0-bb1c-4a7d-b1c2-f967715c964e.png)
展开收起之后Luckysheet宽度不能自适应

**Comment by lxl19920401** [comment]
The width of the Luckysheet is not adaptive after unfolding

**Comment by DONofDESTROY** [comment]
yes, I have kinda fixed this issue by using a hack [react resize dector](https://www.npmjs.com/package/react-resize-detector), by calling luckysheet.resize() whenever the parent div changes size but implementing this on luckysheet would be nice

**Comment by lxl19920401** [comment]
> yes, I have kinda fixed this issue by using a hack [react resize dector](https://www.npmjs.com/package/react-resize-detector), by calling luckysheet.resize() whenever the parent div changes size but implementing this on luckysheet would be nice

The problem is solved. Thank you
